docs-source/content/faq/coherence-requirements.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Coherence Requirements"
+date: 2019-08-12T12:41:38-04:00
+draft: false
+---
+
+If you are running Coherence on Kubernetes, either inside a WebLogic domain
+or standalone, then there are some additional requirements to make sure
+that Coherence can form clusters. 
+
+Note that some Fusion Middleware products, like SOA Suite, use Coherence
+and so these requirements apply to them.
+
+#### Unicast and Well Known Address 
+When the first Coherence process starts, it will form a cluster.  The next
+Coherence process to start (i.e. in a different pod) will use UDP to try
+to contact the master.  Most Kubernetes overlay network providers do not
```

Comment:
Coherence uses the term "senior" for the master node.  It is unrelated to the Kubernetes master (which I am sure you know)